tests/src/python/test_qgsrectangle.py

```diff
@@ -252,6 +252,15 @@ def testToBox3d(self):
         self.assertEqual(box.yMaximum(), 0.3)
         self.assertEqual(box.zMaximum(), 0.5)
 
+    def testOperators(self):
+        rect1 = QgsRectangle(10, 20, 40, 40)
+        rect2 = rect1 + QgsVector(3, 5.5)
+        assert rect2 == QgsRectangle(13, 25.5, 43, 45.5), "QgsRectangle + operator does no work"
+
+        # Subtracting the center point, so it becomes zero.
+        rect1 -= rect1.center() - QgsPointXY(0, 0)
```

Comment:
The intention is to move the rectangle to the zero axis point, so that its center is at (0, 0). You cannot add or subtract points, so I subtract `QgsPoint(0, 0)` to convert QgsPointXY to QgsVector.